scripts/src/checkprcontent/checkpr.py

```diff
@@ -84,6 +84,11 @@ def check_provider_delivery(report_in_pr,num_files_in_pr,report_file_match):
         print(f"::set-output name=providerDelivery::False")
         print(f"[INFO] providerDelivery is a no-go")
 
+def get_file_match_compiled_patterns():
```

Comment:
A docstring explaining what sort of inputs those patterns ingest might clarify the regular expressions below.

Reply:
Yes, and also some description of the captures it might provide, for example:

```python
"""Return a tuple of patterns, where  the first can be used to match a chart PR using any URL, and the second a pattern to match a valid report file within a PR URL.

Both patterns capture ?, ? and ? from the URL.

Examples of valid URLs are:

charts/.../
charts/..../report.yaml
```

Reply:
@isutton I added a docstring based on your input. Hope you like it!